rrbot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100  
  # PID position controller
  joints_position_controller:
    type: effort_controllers/JointGroupPositionController
    joints:
      - joint1
      - joint2
    joint1:
      pid: {p: 1000.0, i: 0.0, d: 3.0}
    joint2:
      pid: {p: 1000.0, i: 0.0, d: 3.0}
